cdap-docs/developers-manual/source/getting-started/dev-env.rst

```diff
@@ -21,24 +21,42 @@ interface methods.
 
 The best way to start developing a CDAP application is by using the Maven archetype:
 
-.. tabbed-parsed-literal::
+.. ifconfig:: snapshot_version
+
+  .. tabbed-parsed-literal::
+  
+    $ mvn archetype:generate \
+        -DarchetypeGroupId=co.cask.cdap \
+        -DarchetypeArtifactId=cdap-app-archetype \
+        -DarchetypeRepository=https://oss.sonatype.org/content/repositories/snapshots/ \
```

Comment:
Well...I have changed the code to now use it. The long story is that in replacements in Sphinx, _you can't have a carriage return in a replacement_ and _you can't have an empty or single space replacement_. So that means I need to different versions of the replacements; one for use where I am splitting the text onto two lines, and one where it is a single line...